packages/axelar-wasm-std/src/utils.rs

```diff
@@ -1,3 +1,6 @@
+use crypto_bigint::U256;
```

Comment:
I think this should be possible to express with existing types

Reply:
You're right. I switched to `alloy-primitives` U256 type, which you already have as a depenendcy.